### ISSUE TITLE
2021-4-24

### ISSUE THREAD

**Issue by ssfc** [ssfc/wangdao#523]
181对抗训练硬核分析：对抗样本与模型参数的关系 https://hub.baai.ac.cn/view/7322 (2021-3-30)
182Successfully run 1_introduction.py; done; (2021-3-30)
183Continue run GCN sample program; dataset.num_classes; Number of classes for each node; (2021-3-30)
184The Cora dataset used in this tutorial only consists of one single graph; A DGL graph can store node features and edge features in two dictionary-like attributes called ndata and edata; (2021-3-30)
185Try to get size of tensor; .shape is OK; (2021-3-30)
186train_mask: A boolean tensor indicating whether the node is in the training set; 
187val_mask: A boolean tensor indicating whether the node is in the validation set; 
188test_mask: A boolean tensor indicating whether the node is in the test set.
189label: The ground truth node category.
